### PR TITLE
1.14.0

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

**Comment by larshp** [comment]
https://rules.abaplint.org/intf_referencing_clas/ now includes a `allow` list, so you can exempt the warning

**Comment by lucasborin** [comment]
image

It`s a possible issue on current existing checks, and it needs further analysis before releasing the new version.

**Comment by lucasborin** [comment]
Issue found on SAP 7.40:
image

File: src/profiles/ytab_checks.tabl.xml [hunk 1]
```diff
@@ -110,34 +94,36 @@
      <COMPTYPE>E</COMPTYPE>
     </DD03P>
     <DD03P>
-     <TABNAME>YTAB_CHECKS</TABNAME>
+     <FIELDNAME>IGNORE_PSEUDO_COMMENTS</FIELDNAME>
+     <KEYFLAG>X</KEYFLAG>
```
Comment (lucasborin) [inline src/profiles/ytab_checks.tabl.xml:98]: if it is key, then, the table needs conversion (SE14). Can we avoid it?

**Comment by lucasborin** [comment]
ABAP 7.40 = OK

**Comment by lucasborin** [comment]
Please, change the changelog.txt date before releasing it.

**Review by eugen-guenther-sap — APPROVED** [review]

**Comment by lucasborin** [comment]
Broken UT:

image

**Pull request merged**